helpers.lua

```diff
@@ -244,8 +244,17 @@ function helpers.sysctl_async(path_table, parse)
 
     spawn.with_line_callback("sysctl " .. path, {
         stdout = function(line)
+	    local separators = {
+	       openbsd = "=",
+	       linux = " = ",
```

Comment:
On GNU/Linux, `sysctl` is usually in `/sbin` and requires superuser permission so I'm not sure about the practical use of it for widgets.

Reply:
> I could use `sysctl` just fine on my Raspberry Pi although the output is very different than the BSD's. I don't know what use it could have either, but it doesn't hurt to have it I guess. :yum:

I obviously forgot that other than desktop computers, GNU/Linux also run on other hardware where permissions might be configured differently.  BTW, the error under GNU/Linux looks like this

    # sysctl foobar
    sysctl: cannot stat /proc/sys/foobar: No such file or directory

Please update the error handling accordingly along with the one on OpenBSD, preferably in the way you handled the separators.

Reply:
I've improved the error handling in a commit. Let me know if that's what you were expecting or what i can change.